strings-csharp/ReadStringFromResourceFile/ReadStringFromResourceFile/Classes/ResourceItem.cs

```diff
@@ -0,0 +1,19 @@
+﻿namespace ReadStringFromResourceFile;
+public class ResourceItem<T>
+{
+    private T? item;
+
+    public ResourceItem(T? item = default(T))
+    {
+        this.item = item;
+    }
+
+    public T? Item { get => item; }
```

Comment:
No need for the `get` here, we can just use the expression body syntax like we are doing on IsValid